google/datalab/contrib/bigquery/commands/_bigquery.py

```diff
@@ -24,6 +24,10 @@ def _create_pipeline_subparser(parser):
                                help='The name of the Composer environment.')
   pipeline_parser.add_argument('-l', '--location', type=str,
                                help='The name of the Cloud location for the environment.')
+  pipeline_parser.add_argument('-d', '--gcs_dag_bucket', type=str,
+                               help='The GCS bucket for the dags for Airflow.')
```

Comment:
GCS -> "Google Cloud Storage" for branding in the help.  flag name is fine.

Reply:
Done.